local-tests/chainrunner/node.py

```diff
@@ -76,14 +80,42 @@ def greplog(self, regexp):
             log = f.read()
         return re.findall(regexp, log)
 
+    def highest_hash(self):
+        highest_block = self.rpc('chain_getBlockHash', None)
+        if isinstance(highest_block, Ok):
+            return highest_block.result
+        else:
+            None
```

Comment:
nit: not sure if `None` without `return` works in python. `None` will still be returned from this fn so just a nit

Reply:
yep, fixed